_includes/header.html

```diff
@@ -17,6 +17,7 @@
 		<link href="css/custom.css" rel="stylesheet" type="text/css" media="all" />
 		<link href="css/blog.css" rel="stylesheet" type="text/css" media="all" />	
 		<link href="css/hover-media-links.css" rel="stylesheet" type="text/css" media="all">
+		<link href="css/backtop.css" rel="stylesheet" type="text/css" media="all">
```

Comment:
This connect back top top button's css with html file

Reply:
Same